include/openssl/dh.h

```diff
@@ -147,7 +147,7 @@ DEPRECATEDIN_3_0(const DH_METHOD *DH_get_default_method(void))
 DEPRECATEDIN_3_0(int DH_set_method(DH *dh, const DH_METHOD *meth))
 DEPRECATEDIN_3_0(DH *DH_new_method(ENGINE *engine))
 
-DH *DH_new(void);
+DEPRECATEDIN_3_0(DH *DH_new(void))
```

Comment:
I don't understand the logic of just deprecating the new..
Why doesnt it also do the free, upref and bits?

Reply:
Because this is WIP and as I said in the introduction:

"There's still more of DH to deprecate, although I think I've solved most problem areas"